cmd/flags.go

```diff
@@ -55,6 +55,7 @@ func initGlobalFlags(globalFlags *pflag.FlagSet) {
 	globalFlags.Bool("bash", false, "Bash flag used to adapt output to bash completion format")
 	globalFlags.MarkHidden("bash")
 	globalFlags.String("context", "", "Name of the context used when connecting to Qlik Associative Engine")
+	globalFlags.BoolP("allow-insecure", "i", false, "Enabling allow-insecure will make it possible to connect using self signed certs")
```

Comment:
In my opinion we should skip the shorthand flag, since it is not recommended to ignore certificate validation (even if we are using self signed certificates. Also change `certs` -> `certificates`